### ISSUE TITLE
Use JSON3

### ISSUE THREAD

**Issue by rafaqz** [JuliaGizmos/WebIO.jl#479]
Type instability in `JSON.Parse` is responsible for a massive amount of load time for most JuliaGizmos packages. It is responsible for about 10 seconds or half of the load time for `Blink.Window()`.

Swapping `AssetRegistry` over to JSON3 more than halved the load time of `Interact.slider`
https://github.com/JuliaGizmos/AssetRegistry.jl/pull/15

It should do something similar here. This probably also means swapping over JSExpr.jl, Knockout.jl and pretty much everything else in JuliaGizmos to use JSON3. I've done most of this, but its a bit of work to finish, so flagging to make sure it can be merged if I do finish it.

**Comment by travigd** [comment]
Unfortunately I think this would be a breaking change since WebIO users can overload the JSON functions to customize their output when serialized. It might be possible to do it in a backwards compatible way (eg fall back to slow JSON), but if not it'd have to be a major version.

**Comment by rafaqz** [comment]
Yes, it would mean moving everything to JSON3 and defining serialization with [StructTypes.jl](https://github.com/JuliaData/StructTypes.jl). But probably everyone should be doing that anyway, it will solve a lot of problems, and the serialization is more generic.

It seems worth it to me - but I'll see how much of that red can be removed first and post another flame graph.

**Comment by rafaqz** [comment]
Also turns out most of this is from JSON.parse calls in Blink.jl, so I'll focus on that for a while and might make some type stability PRs here instead.
